{
  try {
    Assert.assertEquals(StorageResultCode.OperationSucceeded,createContainer("testDestroyContainer",4,"none"));
    Assert.assertEquals(StorageResultCode.OperationSucceeded,destroyContainer("testDestroyContainer",false));
  }
 catch (  Exception e) {
    failStr(e);
  }
}
